addons/hr/views/hr_department_views.xml

```diff
@@ -43,12 +44,15 @@
             <field name="name">hr.department.tree</field>
             <field name="model">hr.department</field>
             <field name="arch" type="xml">
-                <tree string="Companies" sample="1">
+                <tree string="Companies" sample="1" editable="bottom">
+                    <field name="company_id" column_invisible="True"/>
+                    <field name="name"/>
                     <field name="display_name"/>
```

Comment:
Seems a little overkill to have both name and display name here. I guess name had to be added because it's editable but then I think we should just remove display_name?